low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy.sh

```diff
@@ -272,9 +272,10 @@ fi
     # TODO: When RE brings main branch as same strucure as cortx_builds,
     # below should be removed and setup_yum_repo should be able to give cortx-py-utils.
     pkg_name="cortx-py-utils"
-    build_url="http://cortx-storage.colo.seagate.com/releases/cortx/github/main/centos-7.8.2003/cortx-utils_last_successful/"
+    build_url="http://cortx-storage.colo.seagate.com/releases/cortx/components/github/main/centos-7.8.2003/dev/cortx-utils/last_successful/"
+    yum install -y gcc rpm-build python36 python36-pip python36-devel python36-setuptools openssl-devel libffi-devel
```

Comment:
Lot of additional packages pulled here, most of them would be available on cortx build systems already. Why needed for deployment ?

Reply:
removed python36-setuptools, python36 python36-pip.